var/spack/repos/builtin/packages/thrift/package.py

```diff
@@ -45,6 +45,7 @@ class Thrift(Package):
 
     # Variant dependencies
     extends('python', when='+python')
+    depends_on('py-six@1.7.2:', type=('build', 'run'), when='@0.10.0:+python')
```

Comment:
Is this the only Python dependency necessary? In one of the Dockerfiles I see:
```
RUN apt-get install -y --no-install-recommends \
`# Python dependencies` \
      python-all \
      python-all-dbg \
      python-all-dev \
      python-backports.ssl-match-hostname \
      python-ipaddress \
      python-pip \
      python-setuptools \
      python-six \
      python-tornado \
      python-twisted \
      python-wheel \
      python-zope.interface \
      python3-all \
      python3-all-dbg \
      python3-all-dev \
      python3-setuptools \
      python3-six \
      python3-tornado \
      python3-twisted \
      python3-wheel \
      python3-zope.interface && \
    pip install --upgrade backports.ssl_match_hostname
```

Reply:
seems like it. I may have gone overboard with this next push....

Reply:
The `thrift` package doesn't have an official maintainer, so we don't have anyone to ask if these deps are required or optional. Since the `python` variant is optional, I think it's fine to include all deps.